api/src/graphql/schema.ts

```diff
@@ -28,4 +31,6 @@ export const schema = makeSchema({
     export: 'Context',
   },
 });
+
+// @ts-ignore
```

Comment:
Why is it here? Please prefer to [specify exact rules](https://stackoverflow.com/a/43620820/2270613) that needs to be ignored